.github/workflows/check.yml

```diff
@@ -405,3 +405,71 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
       - run: cargo update -w --locked
+
+  check-android:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        target: [x86_64-linux-android, i686-linux-android]
+        emulator-arch: [x86_64, x86]
+        api-level: [26, 25]
+        exclude:
+          - target: x86_64-linux-android
+            emulator-arch: x86
+          - target: i686-linux-android
+            emulator-arch: x86_64
```

Comment:
@mxinden which combinations should we test in CI?

Reply:
@mxinden ping.

Reply:
Oh, sorry for the delay @larseggert. Didn't see this. I don't really have an opinion.

- I think there is value in testing `i686`, not for the platform itself, but for testing a 32bit platform in general.
- API level wise, ideally we use at least the lowest that mozilla-central uses. I am not sure there is value in testing additional ones. Examples of bugs with low API levels are https://github.com/quinn-rs/quinn/pull/1975 and https://bugzilla.mozilla.org/show_bug.cgi?id=1916412.

Reply:
So in other words, what's there currently is OK? :-)

Reply:
Yes. Looking good as is.